src-java/build.gradle

```diff
@@ -56,28 +63,24 @@ subprojects {
             implementation 'com.google.code.gson:gson:2.10.1'
             implementation 'com.fasterxml.uuid:java-uuid-generator:4.0.1'
 
-            implementation('org.apache.kafka:kafka-clients') {
-                version {
-                    strictly '2.3.1'
-                }
-            }
-            implementation('org.apache.kafka:kafka_2.11') {
-                version {
-                    strictly '2.3.1'
-                }
-            }
+//            implementation('org.apache.kafka:kafka-clients') {
```

Comment:
There is a continuation task for this. At this time different modules use different versions. I move them gradually to the new version.